hasura/metadata/databases/default/tables/public_contributions.yaml

```diff
@@ -2,67 +2,52 @@ table:
   schema: public
   name: contributions
 object_relationships:
-  - name: epoch
+  - name: circle
     using:
-      foreign_key_constraint_on: epoch_id
+      foreign_key_constraint_on: circle_id
   - name: user
     using:
       foreign_key_constraint_on: user_id
 insert_permissions:
   - role: user
     permission:
       check:
-        _and:
-          - user:
-              profile:
-                id:
-                  _eq: X-Hasura-User-Id
-          - _or:
-              - epoch_id:
-                  _is_null: true
-              - epoch:
-                  ended:
-                    _eq: false
-      columns:
-        - description
-        - epoch_id
-        - user_id
-      backend_only: false
+        user:
+          profile:
+            id:
+              _eq: X-Hasura-User-Id
+    columns:
+    - circle_id
+    - datetime_created
+    - description
+    backend_only: false
+    set:
+      user_id: x-hasura-User-Id
 select_permissions:
   - role: user
     permission:
       columns:
-        - id
-        - epoch_id
-        - user_id
-        - description
-        - deleted_at
-        - created_at
-        - updated_at
+      - circle_id
+      - created_at
+      - datetime_created
```

Comment:
@topocount `datetime_created` seems weird... how is this different than `created_at` and `updated_at`? `created_at` is when its created and any updates should bump the updated_at time...

Reply:
@crabsinger created_at and updated_at are internal conventions in the db. datetime_created would be a user-modifiable column at some point.

Reply:
Oh, like the date of the contribution, but not the db date.

Reply:
maybe change to `datetime_contributed` ? i think this column is used to identify which epoch the contribution belongs to

Reply:
naming seems abit confusing to me, maybe just  `datetime` or `time` or something that doesnt have `created` on it ? thoughts? @topocount

Reply:
i vote for `contributed_at` to keep the convention that timestamps end in `_at`